### PR TITLE
ENG-5448 Moves Python schema to repo and creates Python package

### PR DESCRIPTION
**Pull request by sebastian-quintero**
# Description

Hello, Python. This PR lays the foundation for working with the `nextroute` Python package, which is an interface for the Go engine.

### CODE REVIEW

File: .github/workflows/release-python.yml [hunk 1]
```diff
@@ -0,0 +1,147 @@
+name: release-python
+run-name: Release ${{ inputs.VERSION }} (pre-release - ${{ inputs.IS_PRE_RELEASE }}) by @${{ github.actor }} from ${{ github.ref_name }}
+
+on:
+  workflow_dispatch:
+    inputs:
+      VERSION:
+        description: "The version to release"
+        required: true
```
Comment (merschformann) [inline .github/workflows/release-python.yml]: Should this derive its version from the Nextroute version it releases, i.e., keep them in sync? (at least for non-testing / pre-release purposes)

File: .github/workflows/release-python.yml
Comment (merschformann) [inline .github/workflows/release-python.yml:1]: For testing this is certainly nicer. I only wonder whether we should combine the two release workflows in a final step (as they are even sharing the repo). See the other comment about syncing version too.

**Review by merschformann — APPROVED** [review]
:tada:

File: .github/workflows/release-python.yml (hunk 1 above)
Comment (sebastian-quintero) [reply .github/workflows/release-python.yml]: Combined workflows into a single release

File: .github/workflows/release-python.yml
Comment (sebastian-quintero) [reply .github/workflows/release-python.yml:1]: Combined into a single release

**Pull request merged**